### ISSUE TITLE
v1 build fail with VCO

### ISSUE THREAD

**Issue by davephillips** [VCVRack/Fundamental#75]
Fedora 23 Linux
VCV Rack v1.dev.6b6c852
GCC 5.3.1

Hit a snag with this morning's pull:

```
g++ -std=c++11 -Wsuggest-override -Idep/include -fPIC -I../../include -I../../dep/include -MMD -MP -g -O3 -march=athlon-fx -funsafe-math-optimizations -Wall -Wextra -Wno-unused-parameter -DARCH_LIN -c -o build/src/VCO.cpp.o src/VCO.cpp
src/VCO.cpp: In member function ‘void VoltageControlledOscillator<OVERSAMPLE, QUALITY, T>::process(float, T)’:
src/VCO.cpp:135:29: error: ‘crossfade’ is not a member of ‘rack::simd’
     triBuffer[i] = 1.129f * simd::crossfade(v0, v1, p);
                             ^
src/VCO.cpp:135:29: note: suggested alternatives:
In file included from ../../include/rack.hpp:5:0,
                 from src/plugin.hpp:1,
                 from src/VCO.cpp:1:
../../include/math.hpp:144:14: note:   ‘rack::math::crossfade’
 inline float crossfade(float a, float b, float p) {
              ^
../../include/math.hpp:144:14: note:   ‘rack::math::crossfade’
../../include/math.hpp:144:14: note:   ‘rack::math::crossfade’
src/VCO.cpp:153:29: error: ‘crossfade’ is not a member of ‘rack::simd’
     sawBuffer[i] = 1.376f * simd::crossfade(v0, v1, p);
                             ^
src/VCO.cpp:153:29: note: suggested alternatives:
In file included from ../../include/rack.hpp:5:0,
                 from src/plugin.hpp:1,
                 from src/VCO.cpp:1:
../../include/math.hpp:144:14: note:   ‘rack::math::crossfade’
 inline float crossfade(float a, float b, float p) {
              ^
../../include/math.hpp:144:14: note:   ‘rack::math::crossfade’
../../include/math.hpp:144:14: note:   ‘rack::math::crossfade’
../../compile.mk:61: recipe for target 'build/src/VCO.cpp.o' failed
make: *** [build/src/VCO.cpp.o] Error 1
```

**Comment by AndrewBelt** [comment]
>`‘crossfade’ is not a member of ‘rack::simd’`

Yes it is. Are you using an old Rack commit?

**Comment by davephillips** [comment]
VCV Rack v1.dev.6b6c852

Am I on the correct branch ?

```
$ git branch
  v0.6
* v1
  v1-gpl
```

**Comment by MarcBoule** [comment]
I was getting this error also, and with the lastest Rack commit 341ae51 and Fundamental commit 270a857 as I write this, the error still persists. Investigating further...

**Comment by AndrewBelt** [comment]
Aha, failed to push. Fixed in 1db6618 of Rack.
